test_cases/search.json

```diff
@@ -156,7 +156,7 @@
     },
     {
       "id": 7,
-      "status": "pass",
+      "status": "fail",
```

Comment:
We will definitely want to add a "description" field here that describes the problem, as well as an "issue" field that links to either a Pelias issue we create, or one in the [Openaddresses](https://github.com/openaddresses/openaddresses/issues) repo if one is there (we should report it there actually). Let's figure out which issue(s) should be made and then add it here.